uuid4s-circe/src/main/scala/org/typepure/uuid4s/implicits/syntax/CirceSyntax.scala

```diff
@@ -0,0 +1,22 @@
+package org.typepure.uuid4s.implicits.syntax
+
+import cats.effect.{IO, LiftIO}
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+import org.typepure.uuid4s.{FUUID, UUID}
+
+trait CirceSyntax extends LiftIOSyntax {
```

Comment:
Because LiftIOSyntax is a trait(not a object). I only intend to add the new behavior to transform effect from IO to Either. Do you prefer that expose LiftIOSyntax as extension (object) and import in this syntax ? For don't depends on it and don't  add unnecessary methods to CirceSyntax.

Reply:
Done